### PR TITLE
[proxy] Update nginx to 1.19.3 and switch to base image to alpine

### PR DESCRIPTION
**Pull request by aledbf**
Switch `nginx 1.14` to `openresty`. Only because it provides a working image with `lua-nginx-module` and a recent nginx version.
(I can also build a custom version, but I don't see any benefit)

### CODE REVIEW

File: chart/config/proxy/nginx.conf [hunk 1]
```diff
@@ -32,22 +30,26 @@ http {
...
-    access_log /dev/stdout mylogfmt;
-    error_log /dev/stderr;
+	log_format mylogfmt escape=none $log_message_json;
+	access_log /dev/stdout mylogfmt;
+	error_log /dev/stderr info;
 
 	##
 	# Gzip Settings
 	##
-	gzip on;
```
Comment (geropl) [inline chart/config/proxy/nginx.conf:45]: On quick scanning this is sth that stood out: Any reason to drop this here, or an unintentional change?

Comment (aledbf) [reply chart/config/proxy/nginx.conf:45]: The include `include /etc/nginx/lib.gzip-assets.conf;` contains the directive + additional setting like min length

**Comment by geropl** [comment]
I wanted to quickly test but got a on login
image

(I assume you're aware :+1:  )

**Review by csweichel — APPROVED** [review]
LGTM

**Pull request merged**